backend/src/main.cpp

```diff
@@ -258,6 +258,29 @@ void SetCid(const Napi::CallbackInfo& info)
     UserSession::cid = cid;
 }
 
+void SetRadioEffects(const Napi::CallbackInfo& info)
+{
+    auto radioEffects = info[0].As<Napi::String>().Utf8Value();
+    auto enableInputFilters = false;
+    auto enableOutputEffects = false;
```

Comment:
It's odd that the value "off" isn't explicitly handled anywhere, and that any unsupported value essentially winds up being "off".

While it technically works, it's probably better to explicitly test for "off" and set the bools to false, and use else-if to enable an else clause at the end that writes a warning to the log file about the value being unsupported and returns without changing anything.

Reply:
Done.